x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields/hooks.tsx

```diff
@@ -135,11 +135,19 @@ export function useOutputOptions(agentPolicy: Partial<NewAgentPolicy | AgentPoli
       getDefaultOutput(defaultOutputName),
       ...outputsRequest.data.items.map((item) => {
         const isInternalOutput = !!item.is_internal;
+        const hasMinimumLicense = licenseService.hasAtLeast(LICENCE_FOR_PER_POLICY_OUTPUT);
 
         return {
           value: item.id,
-          inputDisplay: item.name,
-          disabled: !isPolicyPerOutputAllowed || isInternalOutput,
+          inputDisplay: getOutputLabel(
+            item.name,
+            !hasMinimumLicense ? <FormattedMessage
+                                   id="xpack.fleet.agentPolicyForm.outputOptionDisabledPolicyPerOutput"
+                                   defaultMessage="The current license does not allow defining output per policy."
+                                 />
+            : undefined
+          ),
+          disabled: !hasMinimumLicense || isInternalOutput,
```

Comment:
the `disabled` condition should remain
```suggestion
          disabled: !isPolicyPerOutputAllowed || isInternalOutput,
```